networking/ipv6.md

```diff
@@ -67,13 +67,13 @@ This how-to guide uses the following {{site.prodname}} features:
 #### Enable IPv6 only
 
 {% tabs id:installation-method %}
```

Comment:
Tabs are using two identical `id` which will cause an error for semaphore.

Reply:
Many thanks for the help. Does this id have to be unique as well? <id:op1,name:Operator,active:true>

Reply:
First of all thanks for taking the time to implementing tabs in your doc :).
Yes, `semaphore` uses HTML best practices and panics when theres two identical ID tags in a html document no matter where.
I have addressed this issue in a new PR, hopefully it should be easier to use tabs from now on.

Reply:
Will your new PR eliminate unique ID entirely? Keep me posted because I would like to roll this out to all authors when you believe it is ready.

Reply:
I keep getting error, even though ids are both unique. 

   Generating... 
  Liquid Exception: Liquid syntax error (line 61): Unknown tag 'tabs' in networking/ipv6.md
jekyll 3.8.7 | Error:  Liquid syntax error (line 61): Unknown tag 'tabs'
/usr/gem/gems/liquid-4.0.3/lib/liquid/document.rb:23:in `unknown_tag': Liquid syntax error (line 61): Unknown tag 'tabs' (Liquid::SyntaxError)

Are there line/spacing rules? Do you see any issues with the code? If I open the file, the code looks fine; but the rendering doesn't match what is in the file. ??

Reply:
Yes it will eliminate unique ID being mandatory if you want just have a simple tab system.

Generating...
Liquid Exception: Liquid syntax error (line 61): Unknown tag 'tabs' in networking/ipv6.md
jekyll 3.8.7 | Error: Liquid syntax error (line 61): Unknown tag 'tabs'
/usr/gem/gems/liquid-4.0.3/lib/liquid/document.rb:23:in `unknown_tag': Liquid syntax error (line 61): Unknown tag 'tabs' (Liquid::SyntaxError)

tabs.rb file is missing in plugins folder my best guess.

Reply:
I did a bit of digging and as I suspected `tabs.rb` file is missing in your branch `plugins` folder.
https://github.com/bmckercher123/calico/tree/Enable-IPv6-rewrite/_plugins